packages/eslint-plugin/src/rules/no-useless-template-literals.ts

```diff
@@ -92,21 +93,14 @@ export default createRule<[], MessageId>({
           context.report({
             node: node.expressions[0],
             messageId: 'noUselessTemplateLiteral',
-            fix(fixer): TSESLint.RuleFix[] {
-              const [prevQuasi, nextQuasi] = node.quasis;
-
-              // Remove the quasis and backticks.
-              return [
-                fixer.removeRange([
-                  prevQuasi.range[1] - 3,
-                  node.expressions[0].range[0],
-                ]),
-
-                fixer.removeRange([
-                  node.expressions[0].range[1],
-                  nextQuasi.range[0] + 2,
-                ]),
-              ];
+            fix(fixer): TSESLint.RuleFix | null {
+              const wrappingCode = getMovedNodeCode({
```

Comment:
(nit) the name `wrappingCode` is slightly stale with the rename, but I don't have many good ideas for a better name 🙃. Any ideas? `codeToReplaceTemplate`? Up to you if you want to change this or leave it 🤷‍♂️

Reply:
Since the function name is getMovedNodeCode, how about changing the variable name to movedNodeCode? If you don't think it's appropriate, it might be a good idea to keep the current name.

Reply:
Makes sense to me 👍